src/components/common/dropdown/index.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { FC } from "react";
+import { UseFormRegister, FieldValues, ValidationRule } from "react-hook-form";
+import { GenderType } from "@/state/features/user";
+import styles from "./Dropdown.module.css";
+
+type DropdownProps = {
+  label: string;
+  options: Array<GenderType>;
```

Comment:
Maybe this could be array of strings only if you use it for another thing

Reply:
Fixed!